examples/applications/svm_gui.py

```diff
@@ -138,6 +138,11 @@ def refit(self):
         if self.fitted:
             self.fit()
 
+    def rezoom(self):
+        """Rezoom the window on the scatter plot. """
```

Comment:
what do you mean by rezoom?

Reply:
I mean that the axis limits can be very (too) large after a fit even with points very close to each other.
"Rezoom" will set the axis limits with the min and max of the coordinates of the points (with a little space so that the points are not on the border of the frame).